ocs_ci/ocs/managedservice.py

```diff
@@ -98,6 +98,19 @@ def get_consumer_names():
     return [consumer["metadata"]["name"] for consumer in consumer_yamls]
 
 
+def get_ceph_clients():
```

Comment:
I think that this function should be placed under ocs_ci/ocs/resources/storage_cluster.py as it is not specific to managed service and could be used for converged clusters as well

Reply:
Done